docs/data/date-pickers/custom-field/custom-field.md

```diff
@@ -125,30 +125,30 @@ The new accessible DOM structure will become compatible with Joy UI in the futur
 
 ## With a custom editing experience
 
-### Using an `Autocomplete`
+### Using an Autocomplete
 
 If your user can only select a value in a small list of available dates,
-you can replace the field with an `Autocomplete` listing those dates:
+you can replace the field with the `<Autocomplete />` from `@mui/material` to list those dates:
 
 {{"demo": "PickerWithAutocompleteField.js", "defaultCodeOpen": false}}
 
-### Using a read-only `TextField`
+### Using a read-only Text Field
 
 If you want users to select a value exclusively through the views
-but you still want the UI to look like a `TextField`, you can replace the field with a read-only `TextField`:
+but you still want the UI to look like a TextField, you can replace the field with a read-only `<TextField />` from `@mui/material`:
```

Comment:
```suggestion
but you still want the UI to look like a Text Field, you can replace the field with a read-only Material UI Text Field component:
```

Reply:
I would also like to see the line break removed from this sentence—try to keep it to one sentence per line, even if it's a very long sentence.

Reply:
What is the goal of removing line break on the markdown file?
Those do not impact the line break on the doc..

Reply:
It's just simpler to maintain one sentence per line—if I wanted to edit this sentence more heavily I'd have to leave two separate comments, or leave a suggestion across the two lines. We don't need to add our own line wrapping when an IDE will do that for us.

Reply:
> We don't need to add our own line wrapping when an IDE will do that for us.

Webstorm advises me to add those line breaks though :laughing:

Reply:
Interesting! I don't think we should listen to Webstorm in this case. We don't have a consistent rule to fall back on to decide when to add a line break like this, so I think it would make the docs look really messy if we started doing this everywhere. If we wanted to implement a style rule like this then I think it would need to happen programmatically via Vale or Prettier maybe. I've never used Webstorm before, though. Does it wrap lines for you, or does it display a sentence like this as one long line?

Reply:
> it would make the docs look really messy if we started doing this everywhere

I'm basically doing it everywhere
But if we say that we don't do line break inside a sentence, that's fine for me.

> Does it wrap lines for you, or does it display a sentence like this as one long line?

It just adds a visual feedback, similar to the one when I have a spelling error, I can probably disable it.

Reply:
> But if we say that we don't do line break inside a sentence, that's fine for me.

Let's stick with that then, for the sake of consistency between authors and products.